### PR TITLE
Adding gamepad

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by TheoBoschetti — DISMISSED** [review]
Ok

**Review by TheoBoschetti — APPROVED** [review]
ok

File: frontend/src/Components/Game.jsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Game extends React.Component {
       haveSword: false,
       swordPosition: [{ x: 6, y: 3, swordClass: "Sword" }],
       rubyCounter: 0,
+      gampadConnected: false,
```
Comment (LoicBrassart) [inline frontend/src/Components/Game.jsx:23]: Typo on key name

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**